### DISCUSSION TITLE
Why does this rule not Fire? #solved

### DISCUSSION THREAD

**Discussion by HoerMirAuf** [arendst/Tasmota#11283] [Q&A] [answered]
Maybe anybody can explain this to me?

i have set a clocktimer1

i use this rule to get the clocktime to var1 with console command: "timer1"
`on timer1#time do var1 %value% endon`

This works well but not from another rule:
`on system#boot do timer1 endon`

timer1 will fire, i can see the Result but the trigger timer1#time will now **not** fire.

Where is my misstake?

**Comment by barbudor** [comment, marked as answer]
It occurs that some commands' result do not go through the rule engine when executed through a rule, unless you use backlog
use `on system#boot do backlog timer1 endon`, that should work

**Comment by HoerMirAuf** [comment]
Wow!
Yup, it works with backlog. Thank you very much for this hint !
